sql/views.sql

```diff
@@ -259,5 +259,121 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
+       integer_range_end as range_end_integer,
+       is_compressed,
+       chunk_table_space as chunk_tablespace,
+       node_list as data_nodes
+       from
+(
+SELECT
+    ht.schema_name as hypertable_schema,
+    ht.table_name as hypertable_name,
+    srcch.schema_name as schema_name,
+    srcch.table_name as chunk_name,
+    dim.column_name as primary_dimension,
+    dim.column_type as primary_dimension_type,
+    row_number() over(partition by chcons.chunk_id order by chcons.dimension_slice_id) as chunk_dimension_num,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN _timescaledb_internal.to_timestamp(dimsl.range_start)
+        ELSE NULL
+    END as range_start,
+    CASE
+         WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+                dim.column_type = 'TIMESTAMPTZ'::regtype OR
+                dim.column_type = 'DATE'::regtype )
+         THEN _timescaledb_internal.to_timestamp(dimsl.range_end)
+         ELSE NULL
+    END as range_end,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN NULL
+        ELSE dimsl.range_start
+    END as integer_range_start,
+    CASE
+        WHEN ( dim.column_type = 'TIMESTAMP'::regtype OR
+               dim.column_type = 'TIMESTAMPTZ'::regtype OR
+               dim.column_type = 'DATE'::regtype )
+        THEN NULL
+        ELSE dimsl.range_end
+    END as integer_range_end,
+    CASE WHEN srcch.compressed_chunk_id is not null THEN 'true'
+         ELSE 'false'
+    END as is_compressed,
+    pgtab.spcname as chunk_table_space,
+    chdn.node_list
+FROM _timescaledb_catalog.chunk srcch 
+      INNER JOIN _timescaledb_catalog.hypertable ht ON ht.id = srcch.hypertable_id
+      INNER JOIN _timescaledb_catalog.chunk_constraint chcons ON srcch.id = chcons.chunk_id
+      INNER JOIN _timescaledb_catalog.dimension dim ON srcch.hypertable_id = dim.hypertable_id
+      INNER JOIN _timescaledb_catalog.dimension_slice dimsl ON dim.id = dimsl.dimension_id
+                                and chcons.dimension_slice_id = dimsl.id
+      INNER JOIN
+       ( SELECT relname, reltablespace, nspname as schema_name 
+                FROM pg_class , pg_namespace WHERE
+                pg_class.relnamespace = pg_namespace.oid) cl 
+       ON srcch.table_name = cl.relname and srcch.schema_name = cl.schema_name
+       LEFT OUTER JOIN pg_tablespace pgtab ON pgtab.oid = reltablespace
+       left outer join (
+               SELECT chunk_id, array_agg(node_name ORDER BY node_name) as node_list
+                            FROM _timescaledb_catalog.chunk_data_node
+                            GROUP BY chunk_id) chdn
+       ON srcch.id = chdn.chunk_id 
+ WHERE srcch.dropped is false 
+      and ht.compressed = false ) finalq
```

Comment:
Field `dropped` cannot be null, so:
```suggestion
 WHERE  NOT srcch.dropped AND  NOT ht.compressed ) AS finalq
```